csrankings-h.csv

```diff
@@ -692,6 +692,7 @@ Hong Shen 0001,University of Adelaide,https://cs.adelaide.edu.au/~hong,tBnVrjEAA
 Hong Xu 0001,Chinese University of Hong Kong,https://henryhxu.github.io,BZHzIFIAAAAJ
 Hong Yu 0001,University of Massachusetts Lowell,http://bio-nlp.org,TyXe64wAAAAJ
 Hong Zhang 0013,University of Alberta,https://webdocs.cs.ualberta.ca/~zhang,NOSCHOLARPAGE
+Hong Zhou 0004,Shenzhen University,http://csse.szu.edu.cn/cn/people8a9e.html?53902,Lxr1seAAAAAJ

```

Comment:
Dead home page.

Reply:
Sorry! This is my new home page. 
https://www.researchgate.net/profile/Hong-Zhou-27  
How to update it?

Thanks,
Hong